src/BPEssentials/Events/OnStarted.cs

```diff
@@ -12,6 +12,8 @@ public void OnEvent(SvManager svManager)
             Core.Instance.SvManager = svManager;
 
             Core.Instance.CooldownHandler.Load();
+            Core.Instance.SetupAnnouncer();
+            new Commands.Save().StartSaveTimer();
```

Comment:
Btw we should find a cleaner solution for that someday

Reply:
Right, got any ideas? maybe get the command from the bp command handler?